core/app/models/spree/taxon/paperclip_attachment.rb

```diff
@@ -5,14 +5,14 @@ module Spree::Taxon::PaperclipAttachment
 
   included do
     has_attached_file :icon,
-      styles: { mini: '32x32>', normal: '128x128>' },
-      default_style: :mini,
-      url: '/spree/taxons/:id/:style/:basename.:extension',
-      path: ':rails_root/public/spree/taxons/:id/:style/:basename.:extension',
-      default_url: '/assets/default_taxon.png'
+                      styles: Spree::Config.taxon_image_styles,
+                      default_style: Spree::Config.taxon_image_style_default,
+                      url: '/spree/taxons/:id/:style/:basename.:extension',
+                      path: ':rails_root/public/spree/taxons/:id/:style/:basename.:extension',
+                      default_url: '/assets/default_taxon.png'
 
     validates_attachment :icon,
-      content_type: { content_type: %w[image/jpg image/jpeg image/png image/gif] }
+                         content_type: { content_type: %w[image/jpg image/jpeg image/png image/gif] }
```

Comment:
I think we can replace the array with  `Spree::Config.allowed_image_mime_types` here

Reply:
Thanks for catching this, I agree that it should have been changed

Reply:
Apparently it is breaking the application, I am not sure why yet

Reply:
I didn't try it, but the correct format should be  `content_type: { content_type: Spree::Config.allowed_image_mime_types }`